src/content/apply/developer.md

```diff
@@ -11,9 +11,9 @@ mailLink: "https://gmail.us3.list-manage.com/subscribe?u=3b3ae33f54203ab7a839ae5
 role: Developer
 description: "We're looking for developers to work on research projects with researchers and professors as well as community projects for the Northeastern student body. We have frontend, backend, and full stack developers who are passionate about contributing to the community. As Sandbox continues to grow, we’re building a diverse and skilled team of developers with a variety of experiences, interests, and backgrounds to make amazing software with us."
 isVisible: true
-isOpen: true
-openDate: ""
-closeDate: "May 30th 2021"
+isOpen: false
+openDate: "November 29th 2021"
+closeDate: ""
```

Comment:
do we mentioned close date as well? currently, we have it planned to be closed initially on Dec. 6th

Reply:
ya I originally put that there but then it felt weird to put the close date when the app isn't even open yet. Ill send you a pic